openpype/hosts/maya/api/pipeline.py

```diff
@@ -426,6 +479,46 @@ def on_before_save():
     return lib.validate_fps()
 
 
+def check_lock_on_current_file():
+
+    """Check if there is a user opening the file"""
+    if not handle_workfile_locks():
+        return
+    log.info("Running callback on checking the lock file...")
+
+    # add the lock file when opening the file
+    filepath = current_file()
+
+    if is_workfile_locked(filepath):
+        # add lockfile dialog
+        from Qt import QtWidgets
+        top_level_widgets = {w.objectName(): w for w in
+                             QtWidgets.QApplication.topLevelWidgets()}
+        parent = top_level_widgets.get("MayaWindow", None)
+        workfile_dialog = WorkfileLockDialog(filepath, parent=parent)
```

Comment:
I think it will work even without the parent. If it does not then please use `get_main_window` from `.lib` to get it.
```suggestion
        workfile_dialog = WorkfileLockDialog(filepath)
```

With parent
```
        parent = get_main_window()
        workfile_dialog = WorkfileLockDialog(filepath, parent=parent)
```